test/test_contact_importer.rb

```diff
@@ -20,10 +15,10 @@ def test_u_p_import
   end
 
   def test_aol_import
-    VCR.use_cassette("aol_import", :match_requests_on => [:uri, :host, :path]) do
+    VCR.use_cassette("aol_import", :match_requests_on => [:method, :host, :path]) do
       importer = Cloudsponge::ContactImporter.new(DOMAIN_KEY, DOMAIN_PASSWORD)
       resp = importer.begin_import('AOL')
-      if resp[:consent_url]
+      if ENV['request_consent'] && resp[:consent_url]
```

Comment:
it is an environment variable to pass in on the command line. by default I
want to suppress the popup window because consent is being simulated by
VCR.
On Wed, Apr 13, 2016 at 7:58 AM Alfredo Cipriani Oliveira Ribeiro <
notifications@github.com> wrote:

> In test/test_contact_importer.rb
> <https://github.com/cloudsponge/cloudsponge-lib-ruby/pull/3#discussion_r59563876>
> :
>
> >        importer = Cloudsponge::ContactImporter.new(DOMAIN_KEY, DOMAIN_PASSWORD)
> >        resp = importer.begin_import('AOL')
> > -      if resp[:consent_url]
> > +      if ENV['request_consent'] && resp[:consent_url]
>
> Nervermind, I get it, you can set in the tests to request consent or not,
> thanks!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudsponge/cloudsponge-lib-ruby/pull/3/files/d449b957024d57b531afd68fb38fb3881014fe42..5680eb4caaec6d6c0c0d05de97db86e85a493eee#r59563876>
>